src/System.Memory/ref/System.Memory.cs

```diff
@@ -87,6 +87,7 @@ public static class SpanExtensions
     {
          public static int IndexOf<T>(this Span<T> span, T value) where T:struct, IEquatable<T> { throw null; }
```

Comment:
Yeah, I agree corfxlab might be a better place for it. corfx APIs should represent what we are confident in shipping.